buildNumber.properties

```diff
@@ -1,3 +1,3 @@
 #maven.buildNumber.plugin properties file
-#Tue Dec 12 15:37:44 CST 2017
-buildNumber=40
+#Wed May 01 22:38:17 BST 2019
+buildNumber=46
```

Comment:
This file should be removed from the pull request.
In the future we want to remove it.

Reply:
ok will revert it.